src/main/java/com/epam/training/tasks/third/data/PointDataValidator.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.training.tasks.third.data;
+
+public class PointDataValidator {
+
+    public boolean validate (double[] data) {
+
```

Comment:
no extra empty lines, you them to split logical blocks, not evetywhere, otherwise they are not useful